templates/base/src/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Router from 'next/router';
+import getConfig from 'next/config';
+
+const {
+  publicRuntimeConfig
+} = getConfig();
+
+export const BASE_PATH = publicRuntimeConfig.basePath;
+
+export const NextRouterWrapper = ({
+  pathName,
+  as,
+  options = {}
+}) => {
+  Router.push(pathName, `${BASE_PATH}${as || pathName}`, options);
+};
+
+export const getBasePath = route =>
+  `${BASE_PATH}${route}`
+
+export default {
```

Comment:
We should maintain this in a separate file and import and export it here. We will be auto updating it with PLP